site/diagrams/sequenceDiagrams/command_control_concept_mqtt.txt

```diff
@@ -0,0 +1,18 @@
+participant Device
+participant MQTT Adapter
+participant AMQP 1.0 Network
+participant Application
+
+Device->MQTT Adapter: 1. send event
+note over Device
+content-type=application/vnd.eclipse-hono-empty-notification
+-> MQTT Adapter will convert this in tdd=-1 (infinitely)
+end note over
+activate MQTT Adapter
+MQTT Adapter->Application: 2. connected notification (ttd=-1)
```

Comment:
this is simply the transfer of the (empty) event ...

Reply:
Hmm - yes, and I shown it differently in the both diagrams. I will rework this - but I need to make it visible, that the ttd header is the relevant part.